faq.md

```diff
@@ -1,45 +1,634 @@
 
-#### What kind of abstraction does OpenTK provide?
-The Open Toolkit mirrors the raw OpenGL API, with the addition of type-safe enums. This was a conscious design decision that ensures high performance, easy translation of C/C++ code, and beyond that, matches the OpenTK philosophy of being a toolkit, rather than a framework or engine.
+#### What is OpenCL?
 
-#### How do I use this?
-1. Add the NuGet Package to your project
-2. Open the appropriate namespace
-3. Start coding
+[OpenCL](https://en.wikipedia.org/wiki/OpenCL) is a cross-platform parallel-computation specification, originally developed by [Apple](https://www.apple.com), and now maintained by the [Khronos group](https://www.khronos.org).  It’s primarily used for performing computation on GPUs that would otherwise be computed less efficiently on CPUs, but it can also be used to control digital signal processors (DSPs), field-programmable gate arrays (FPGAs), and other kinds of hardware compute accelerators.
 
-#### I need help!
-Try asking on our [Discord server](https://discord.gg/6HqD48s).
+#### What is OpenAL?
 
-[![Discord](https://discordapp.com/api/guilds/337627185248468993/widget.png)](https://discord.gg/6HqD48s)
+[OpenAL](https://openal.org/) is a cross-platform 3D audio library, originally developed by [Loki Software](https://en.wikipedia.org/wiki/Loki_Software), and released as an open specification by [Creative Labs](https://us.creative.com/), with multiple implementations in both hardware and software.  OpenAL is used in lots of games to provide both sound effects and music and is the successor to EAX and A3D.
 
-#### Something is broken!
-[Open an issue at the repo](https://github.com/opentk/opentk/).
+### What operating systems are supported by OpenTK?
 
-#### Who are the maintainers?
-[@thefiddler](https://github.com/thefiddler) is the original author.
-Current maintainers are [@frederikja163](https://github.com/frederikja163), [@jvbsl](https://github.com/jvbsl), [@PJB3005](https://github.com/PJB3005), and [@varon](https://github.com/varon).
-Previous maintainers include [@Nihlus](https://github.com/Nihlus), and [@Perksey](https://github.com/Perksey)
+OpenTK runs on Windows, Linux, and MacOS X.
 
-#### How can I help?
-Check outstanding issues, or drop us a message in Discord.
+(Older versions of OpenTK supported Android and iOS as well; current OpenTK *might* work on these, but it’s untested.)
+
+### What hardware is supported?
+
+OpenTK supports any hardware that is supported by your OS’s native OpenGL, OpenCL, and OpenAL bindings:  Since OpenTK is just a .NET layer around the native libraries, it does whatever they do.
+
+### Which .NET versions are supported?
+
+- **OpenTK 4.x+** runs on .NET Core 2.x, .NET Core 3.x, and .NET 5+.
```

Comment:
OpenTK 4 only runs on .net core 3.1+ iirc.

Reply:
I don't know that for a fact, but I've removed .NET Core 2.x from the list anyway; anyone using .NET Core is likely on a newer version of it anyway and isn't likely going to be ask why .NET Core 2.x isn't supported.